### PR TITLE
docs: add section about articles about finetuner

### PR DESCRIPTION
**Pull request by guenthermi**
Add section about articles about finetuner
---

just add a section to the Readme and the docs which reference the blog posts about finetuner. If you have more to reference, let me know.

- [ ] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,18 @@ pip install "finetuner[full]"
 > ⚠️ Starting with version 0.5.0, Finetuner computing is performed on Jina AI Cloud. The last local version is `0.4.1`. 
 > This version is still available for installation via `pip`. See [Finetuner git tags and releases](https://github.com/jina-ai/finetuner/releases).
 
+<!-- start finetuner-articles -->
+## Articles about Finetuner
+
+There are several articles about Finetuner and tutorials which using our framework:
```
Comment (bwanglzu) [inline README.md:168]: i would rephrase it a bit as something like: we have created several articles, all blog posts below are our "internal" blog post

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (guenthermi) [reply README.md:168]: yes you are right, first I thought we can add some external as well but I did not found good ones to add.

File: README.md [hunk 2]
```diff
@@ -162,6 +162,18 @@ pip install "finetuner[full]"
 > ⚠️ Starting with version 0.5.0, Finetuner computing is performed on Jina AI Cloud. The last local version is `0.4.1`. 
 > This version is still available for installation via `pip`. See [Finetuner git tags and releases](https://github.com/jina-ai/finetuner/releases).
 
+<!-- start finetuner-articles -->
+## Articles about Finetuner
```
Comment (gmastrapas) [inline README.md:166]: Blogposts?

File: README.md [hunk 3]
```diff
@@ -162,6 +162,18 @@ pip install "finetuner[full]"
...
+
+We wrote several articles and tutorials in which our framework is used:
```
Comment (gmastrapas) [inline README.md:168]: Check out our published blogposts and tutorials to see Finetuner in action!

**Review by gmastrapas — APPROVED** [review]
